compositor_api/src/types/from_util.rs

```diff
@@ -54,6 +54,7 @@ impl TryFrom<Transition> for scene::Transition {
         Ok(Self {
             duration: Duration::from_secs_f64(transition.duration_ms / 1000.0),
             interpolation_kind,
+            reset_on_update: transition.reset_on_update.unwrap_or(false),
```

Comment:
```suggestion
            can_interupt: transition.reset_on_update.unwrap_or(false),
```
rename in other place too